webservices/resources/reports.py

```diff
@@ -105,11 +121,27 @@ def build_query(self, committee_id=None, committee_type=None, **kwargs):
         if kwargs.get('is_amended') is not None:
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
+        query = filters.filter_range(query, kwargs, self.filter_range_fields)
+
         return query, reports_class, reports_schema
 
     def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwargs):
-        if committee_id is not None:
-            query = models.CommitteeHistory.query.filter_by(committee_id=committee_id)
+        #This could be a potential pit fall, is there a better way to get
+        #committee type?  Was the reason this was done this way as a committee
+        #type can change over time?  Also take strong note
```

Comment:
Committee type can change over time. Authorized committees frequently become PACs when the candidate is no longer in the race,